ISSUE_TEMPLATE.md

```diff
@@ -43,12 +43,17 @@ https://github.com/constellation-app/constellation/blob/master/CODE_OF_CONDUCT.m
 
 3.  \[and so on…\]
 
+### Behaviour
+
+Try to include measurable metrics that can be aimed for (if applicable to the issue),
+**e.g.** If its a time issue then "should take 5 seconds or less to finish"
+or if its a workflow issue "should be accomplished in 2 or less clicks".
```

Comment:
I would comment this out (i.e. surround it in \<!-- Try to include...  -->).

Also "it's a time" and "it's a workflow"